provisions/roles/db/tasks/main.yml

```diff
@@ -37,13 +37,6 @@
   sudo: yes
   systemd: name=docker state=started enabled=yes
 
-- name: Pull CentOS postgres container
-  docker_image:
-      name: registry.centos.org/sclo/postgresql-95-centos7
```

Comment:
I would suggest we replace line no 42, the hardcoded name of `sclo` namespaced image with `'{{ postgresql_image }}'`.
Thus, there is `pull` and `run` tasks.

Reply:
Agree. But we don't need to explicitly do it since that's taken care of [here](https://github.com/CentOS/container-pipeline-service/blob/22dd7b52edaa0f5ab9e2fba8c89c262ce9e35b67/provisions/roles/db/tasks/main.yml#L47-L62). `docker_container` module pulls an image if one it's supposed to create container out of doesn't exist.

Reply:
I know that. I wanted to convey about dividing pull and run between two tasks for better control. Like if you can update the pull image task with `force: yes`, it will ensure that always there is an effort to pull (which will ensure during every deployment image is upto date) before run.

Reply:
I had same concern for using an up-to-date image during each deployment. But that warrants a different PR, right? wdyt?

Reply:
I think we can just remove the hardcoded `sclo` repository path and use the variable. Add another commit to same PR, saying add force pull image on every deployment, update PR title, we are good!

Reply:
sounds good. made changes. PTAL.